testing/src/test/java/org/motechproject/nms/testing/it/kilkari/SubscriptionServiceBundleIT.java

```diff
@@ -760,4 +762,22 @@ public void testDeleteOldCompletedSubscription() {
         subscriber = subscriberDataService.findByCallingNumber(2000000000L);
         assertEquals(1, subscriber.getSubscriptions().size());
     }
+    
+    @Ignore
+    @Test
+    public void verifyFT153() {
+    	/*
```

Comment:
Hi Koshal,

Doesn't it look cleaner to have test case description ( pasted from test
case doc) just inside the method ?
It looked to me quite good when @Frank and I pasted it inside the method
body.

On Tue, Jun 9, 2015 at 11:08 PM, kosh <notifications@github.com> wrote:

> In
> testing/src/test/java/org/motechproject/nms/testing/it/kilkari/SubscriptionServiceBundleIT.java
> <https://github.com/motech-implementations/mim/pull/352#discussion_r32040948>
> :
>
> > @@ -760,4 +762,22 @@ public void testDeleteOldCompletedSubscription() {
> >          subscriber = subscriberDataService.findByCallingNumber(2000000000L);
> >          assertEquals(1, subscriber.getSubscriptions().size());
> >      }
> > +
> > +    @Ignore
> > +    @Test
> > +    public void verifyFT153() {
> > +    	/*
>
> @AnkitGaurNms <https://github.com/AnkitGaurNms> @PriyaJain5
> <https://github.com/PriyaJain5> @kanchannms
> <https://github.com/kanchannms> @srivastavaAbhi
> <https://github.com/srivastavaAbhi> Please make sure to add the comment
> for the method on top and not inside. I understand @frankhuster
> <https://github.com/frankhuster> might have started this trend
> originally, but we should stick with the convention and move the comments
> to the top of the method.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/motech-implementations/mim/pull/352/files#r32040948>.
>